lib/screens/home_screen/file_explorer_section.dart

```diff
@@ -128,8 +130,28 @@ class FileExplorerSection extends StatelessWidget {
     );
   }
 
-  void _openInVSCode(FileSystemEntity entity) {
-    Process.start("code", [entity.path], runInShell: true);
+  void _openInVSCode(FileSystemEntity entity, BuildContext context) {
+    try {
+      if (Platform.isWindows) {
+        Process.start("code.exe", [entity.path], runInShell: true);
+      } else if (Platform.isMacOS) {
+        Process.start(
+            "/Applications/Visual Studio Code.app/Contents/Resources/app/bin/code",
+            [entity.path],
+            runInShell: true);
+      } else if (Platform.isLinux) {
+        Process.start("code", [entity.path], runInShell: true);
```

Comment:
While utilizing a package such as 'process_run' can simplify the path-finding process, I appreciate this method for its straightforwardness. We can address any errors if they arise later on.